core/src/main/res/values-de/strings.xml

```diff
@@ -285,6 +286,8 @@
   <string name="ascending">Aufsteigend</string>
   <string name="descending">Absteigend</string>
   <string name="clear_queue_confirmation_msg">Bitte bestätige, dass ALLE Episoden aus der Abspielliste entfernt werden sollen</string>
+  <string name="sort_new_to_old">Neu bis alt</string>
+  <string name="sort_old_to_new">Alt bis neu</string>
```

Comment:
Translations are managed by Transifex and are overwritten when changed in the language specific files

Reply:
Ok, thank you for the hint.